test/options/__snapshots__/dynamicPublicPath.test.js.snap

```diff
@@ -0,0 +1,3 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Options dynamicPublicPath {Function} 1`] = `"throw new Error(\\"Module parse failed: /Users/anvar/projects/quanttech/file-loader/src/index.js??ref--0!/Users/anvar/projects/quanttech/file-loader/test/fixtures/file.png Identifier directly after number (1:23)\\\\nYou may need an appropriate loader to handle this file type.\\\\n| module.exports = test/9c87cbf3ba33126ffd25ae7f2f6bbafb.png;\\");"`;
```

Comment:
There is something off as the loader throws an `{Error}`

Reply:
Thanks for the feedback! Sure thing, the main thing is that `publicPath` takes the return value from the function and wraps it in `JSON. stringify`, which means that you cannot use variables that are to be resolved during runtime in the application. An example is `window.SomeVariable`.

In regards to the test, it is a bit odd. It did the same when I ran it locally, the I ran `npm test -- -u` as suggested by the script after which it worked flawlessly.